### PR TITLE
PhpUnitNoExpectationAnnotationFixer - fix handling of multiline expectedExceptionMessage annotation

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: tests/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixerTest.php [hunk 1]
```diff
@@ -461,6 +461,40 @@ public function testElementNonExistentTwo()
...
+<<<'EOT'
+    <?php
+    final class MyTest extends \PHPUnit_Framework_TestCase
+    {
+        /**
+         * @expectedException \RuntimeException
+         * @expectedExceptionMessage Message
+         *                           on
+         *                           multilines
+         */
```
Comment (keradus) [inline tests/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixerTest.php]: please add cases like
```
 +        /**
 +         * @expectedException \RuntimeException
 +         * @expectedExceptionMessage Message
 +         *                           on
 +         *                           multilines
 +         * @foo
 +         */
```
and
```
 +        /**
 +         * @expectedException \RuntimeException
 +         * @expectedExceptionMessage Message
 +         *                           on
 +         *                           multilines
 +         *
 +         * @foo
 +         */
```

Comment (Slamdunk) [reply tests/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixerTest.php]: Done

**Comment by Slamdunk** [comment]
@keradus `PhpUnitNoExpectationAnnotationFixer` source seems not so trivial, if you could take care of the bugfix I'd appreciate it. Btw the priority to me is very low, near zero, just found the case in a project and wanted to report the bug.

**Comment by Slamdunk** [comment]
Never mind, just found the time to fix it 😛

**Comment by keradus** [comment]
Thank you @Slamdunk.

**Pull request closed without merging**